exporter/signalfxexporter/internal/dimensions/metadata.go

```diff
@@ -44,10 +45,14 @@ func sanitizeProperty(property string) string {
 	return property
 }
 
-func getPropertiesAndTags(kmu metadata.MetadataUpdate) (map[string]*string, map[string]bool) {
+func getPropertiesAndTags(defaults map[string]string, kmu metadata.MetadataUpdate) (map[string]*string, map[string]bool) {
 	properties := map[string]*string{}
 	tags := map[string]bool{}
 
+	for k, v := range defaults {
+		properties[k] = &v
```

Comment:
Do we want to validate that value is not empty string ? As keys with empty values are treated as tags here, or maybe shall we also support tags?

Reply:
Thanks, good call - added.